data/transactions/logic/doc_test.go

```diff
@@ -55,11 +55,11 @@ func TestDocStragglers(t *testing.T) {
 
 	for op := range opDocExtras {
 		_, ok := opDocByName[op]
-		require.True(t, ok, "%s is in opDocExtra, but not opDocByName", op)
+		assert.True(t, ok, "%s is in opDocExtra, but not opDocByName", op)
```

Comment:
so I suppose you want to carry on the test, even if non-existing opcode's prev doc is still out there?  I suppose this is helpful in debugging for later deprecations...

Reply:
Yeah, I noticed that it was nice to get all the reports at once, when modifying a bunch of opcodes.